AppController/terminate.rb

```diff
@@ -1,5 +1,5 @@
 $:.unshift File.join(File.dirname(__FILE__), "lib")
-APPSCALE_HOME = ENV['APPSCALE_HOME']
+CONFIG_FILE_LOCATION = "/etc/appscale"
```

Comment:
Roger that: I change the variaable to APPSCALE_CONFIG_DIR everywhere I found it.